src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,17 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(val ciudadDeOrigen: Ciudad, val vendedores: MutableList<Vendedor>) {
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        } else {
+            error("El vendedor ya está registrado en el centro de distrubición.")
+        }
```

Comment:
Se lee un poco mejor si chequeás el error primero y no ponés el `else`:
```suggestion
        if (vendedores.contains(vendedor)) {
            error("El vendedor ya está registrado en el centro de distrubición.")
        }
        vendedores.add(vendedor)
```